setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='nsaph',
-    version="0.0.4.6",
+    version="0.0.4.7",
```

Comment:
I am familiar with github releases for stamped releases, is this an alternative?